silverback/_click_ext.py

```diff
@@ -242,3 +244,16 @@ def get_cluster_client(ctx: click.Context, *args, **kwargs):
         return ctx.invoke(f, *args, **kwargs)
 
     return update_wrapper(get_cluster_client, f)
+
+
+def bot_path_callback(ctx: click.Context, param: click.Parameter, value):
+    path = ctx.params.get("path")
```

Comment:
See previous, you don't need this

```suggestion
```

And you also won't need the additional `path` argument to your CLI, you can use it just like

```py
@click.argument("bot", required=False, callback=...)
```

Reply:
If a user gives us a custom path, wouldn't we need this?

Reply:
the callback should process the path and return a module import